data/mil2525c_b2/test_data/Complete_test_scripts/README.md

```diff
@@ -0,0 +1,17 @@
+# military-features-data / data / mil2525c_b2 / test_data / Complete_test_scripts
+==========================
+
+# Purpose 
+
+The script and model in this folder automates the workflow for creating comprehensive test datasets for 2525 Military Standards.
+
+## Instructions 
+
+* Connect the `Complete_test_scripts` folder to one of the Military Overlay Information Models for ArcGIS Pro.
+* Open the `FilterCSVMilSymbols` Script. For the File to Filter parameter, use any 2525 source .csv, and the script will create output .csv files filtered by their appendix. For example, the source .csv files for 2525 C and Bc2 can be found [here](https://github.com/Esri/military-features-data/tree/v.next/data/mil2525c_b2/test_data/truth_data). 
+* Choose an appropriate output folder and run the script. 
```

Comment:
You normally want to use a relative link ex: `"./truth_data"` so the link still works if cloned/changed - may not matter in this case if that folder/link never changes